test/e2e-encryption/encryption_test.go

```diff
@@ -6,22 +6,29 @@ import (
 
 	"github.com/stretchr/testify/require"
 
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/util/rand"
 
 	configv1 "github.com/openshift/api/config/v1"
-	"github.com/openshift/cluster-kube-apiserver-operator/test/library/encryption"
+	"github.com/openshift/cluster-kube-apiserver-operator/pkg/operator/operatorclient"
+	operatorencryption "github.com/openshift/cluster-kube-apiserver-operator/test/library/encryption"
+	"github.com/openshift/library-go/test/library/encryption"
 )
 
 func TestEncryptionTypeIdentity(t *testing.T) {
```

Comment:
in `step2` I will move test cases into `library-go`, it seems like they can be reused by `kube` and `openshift`